assets/models/system/bert-base-cased/description.md

```diff
@@ -31,12 +31,14 @@ Fill Mask|Fill Mask|<a href="https://huggingface.co/datasets/rcds/wikipedia-for-
 #### Sample input
 ```json
 {
-    "inputs": {
```

Comment:
Same as gpt2 PR. Please either include all models changes in one PR and close remaining PRs or segregate to keep only the changes specific to a model mentioned in PR title